microsoft-edge/webview2/concepts/windowed-vs-visual-hosting.md

```diff
@@ -6,17 +6,65 @@ ms.author: msedgedevrel
 ms.topic: conceptual
 ms.service: microsoft-edge
 ms.subservice: webview
-ms.date: 10/24/2022
+ms.date: 09/18/2024
 ---
 # Windowed vs. visual hosting of WebView2
 
-There are two options for hosting the Microsoft Edge WebView2 control in your app: windowed hosting and visual hosting.
+<!-- outline Sep 28, 2024:
```

Comment:
inspect outline re: the many headings that are APIs Ref docs tabsets - maybe add "(APIs)" to those headings.  
Inspect the structure of the article -- seems like a ton of APIs sections are placed unpredictably, and need to be mentioned or introduced as API Ref sections.

Reply:
solution: added same-level "APIs" headings (in article body & hidden outline):
```
=======================================
## Windowed hosting and Window to Visual hosting
#### Benefits of Windowed hosting
#### Benefits of Window to Visual hosting
#### APIs for Windowed hosting and Window to Visual hosting
#### Window management                            [APIs tabset]
...

=======================================
## Visual hosting
#### APIs for Visual hosting
#### Composition-based rendering                  [APIs tabset]
...
```

Reply:
updated the 1st anchor link reported by build report:
```
These approaches have different requirements, constraints, and benefits.  Windowed hosting is simpler to implement than Visual hosting.  Visual hosting requires all the API calls that Windowed hosting requires, and Visual hosting has additional requirements for it to render properly.  The API calls are listed in the following sections below:
* [Windowed hosting and Window to Visual hosting](#windowed-hosting-and-window-to-visual-hosting)
* [Visual hosting](#visual-hosting)
```